<launch>
    <arg name="duckiebot" default="duckiebot" />

    <arg name="cam_frame" default="camera" doc="tf frame id for camera frame" />
    <arg name="vehicle_frame" default="car" doc="tf frame id for vehicle frame"/> 
    <arg name="world_frame" default="world" doc="tf frame id for world frame"/>

    <rosparam command="load" file="$(find duckietown-intnav)/config/duckiebot.yaml" ns="$(arg duckiebot)/params" />

    <node pkg="duckietown-intnav" type="april_tf_static.py" name="tf_april_static" output="screen" >
        <param name="duckiebot" value="$(arg duckiebot)" />
        <param name="world_frame" value="$(arg world_frame)" />
        <param name="vehicle_frame" value="$(arg vehicle_frame)" />
        <param name="cam_frame" value="$(arg cam_frame)" />
    </node>

    <!-- Useful ressource: https://quaternions.online/ -->
</launch>
